### ISSUE TITLE
Unable to Create New Resolution (SQLSTATE[22003]: Numeric value out of range: 1264 Out of range value for column 'userId' at row 1)

### ISSUE THREAD

**Issue by otdi-gh** [xibosignage/xibo#3212]
Hello Xibo Team,

We noticed a bug in self-hosted docker CMS v3.3.5+ (and likely prior versions as well). This bug prevents users with a userId > 127 from being able to create a new Resolution in the CMS. 

Attempting to create a new resolution from an account with userId > 127 results in the following:

![image](https://github.com/xibosignage/xibo/assets/135650772/c02802e8-1623-4f7e-a054-ae0c3ec7bf5e)

The resultant error from the attempt in the logs is:
`SQLSTATE[22003]: Numeric value out of range: 1264 Out of range value for column 'userId' at row 1`

`mysql> describe resolution;`
`+-----------------+--------------+------+-----+---------+----------------+`
`| Field           | Type         | Null | Key | Default | Extra          |`
`+-----------------+--------------+------+-----+---------+----------------+`
`| resolutionID    | int(11)      | NO   | PRI | NULL    | auto_increment |`
`| resolution      | varchar(254) | NO   |     | NULL    |                |`
`| width           | smallint(6)  | NO   |     | NULL    |                |`
`| height          | smallint(6)  | NO   |     | NULL    |                |`
`| intended_width  | smallint(6)  | NO   |     | NULL    |                |`
`| intended_height | smallint(6)  | NO   |     | NULL    |                |`
`| version         | tinyint(4)   | NO   |     | 1       |                |`
`| enabled         | tinyint(4)   | NO   |     | 1       |                |`
`| userId          | tinyint(4)   | NO   |     | NULL    |                |`
`+-----------------+--------------+------+-----+---------+----------------+`

**_Note the type for the field userId: tinyint(4)._**

Per signed tinyint specs, the only values allowed for userId in this table would be -127 thru +127. 

To verify, we had userId=2 attempt to create the same Resolution with complete success.

Steps to Replicate:

1. Manually set userId to value > 127 in database 
_(Note: The correct way to 'replicate' this would be to populate your CMS with 127 test accounts and then proceed with an account that has a userId > 127)_
2. Sign into the CMS with account
3. Attempt to create a new resolution

Please let us know if you have any questions.

Thanks!

**Comment by dasgarner** [comment]
Thank you for reporting this issue.

I've checked in the installation script for 3.3 and 4.0 and can confirm that userId column on resolution is an integer. In 1.7 the resolution table didn't have a userId column; it was added in the upgrade to 1.8 using the following SQL:

```
ALTER TABLE `resolution` ADD userId INT NOT NULL;
```

As far as I can see from version history, it was always added in that way.

So the bottom line here is I don't know how your `userId` column got set to tinyint. I think the best thing to do is for you to manually update it.

**Comment by otdi-gh** [comment]
@dasgarner 
Thank you for looking into this issue. However, I'm confused at the dismissal of the bug report. We have a dev CMS that we've upgraded in parallel since v1.8. We've made no custom modifications to it and have verified that it too is affected, as seen in the screenshot below:

<img width="1244" alt="image" src="https://github.com/xibosignage/xibo/assets/135650772/39efda2d-ef5b-4b35-abec-f3029eea3420">

I assure you that somewhere along the line the database table was altered to TINYINT.
